docs/index.rst

```diff
@@ -10,7 +10,7 @@ meson-python
 .. highlights::
 
    ``meson-python`` is a Python build backend built on top of the Meson_
```

Comment:
I would remove the link also from the first mention of Meson in the colophon. It really does not look nice. I would maybe link the first mention of Meson in the text.

Reply:
Agreed. Done now. That first mention is in the intro tutorial rather than on this page, but that's fine.